code/game/objects/items.dm

```diff
@@ -1,658 +1,681 @@
-/obj/item

```

Comment:
This whole file was deleted during one of the master merges, for reasons I do not understand. It **should** be more or less a clone of what it is at present, with two procs added and the definition for /obj/item/device moved to the bottom of the file.